File: Trustify.Admin/src/app/api/models/user-wrapper.ts
/* tslint:disable */
import { Credentials } from './credentials';
export interface UserWrapper {
  credentials?: null | Array<Credentials>;
  email?: null | string;
  emailVerified?: null | boolean;
  enabled?: null | boolean;
  firstName?: null | string;
  lastName?: null | string;
  requiredActions?: null | Array<string>;
  userName?: null | string;
}

File: Trustify.Backend/Trustify.Backend.AdminService/Models/UserWrapper.cs
using Newtonsoft.Json;
using Trustify.Backend.AdminService.Keycloak.Models;

namespace Trustify.Backend.AdminService.Models
{
    public class UserWrapper
    {
        [JsonProperty("username")]
        public string? UserName { get; set; }

        [JsonProperty("enabled")]
        public bool? Enabled { get; set; }

        [JsonProperty("emailVerified")]
        public bool? EmailVerified { get; set; }

        [JsonProperty("firstName")]
        public string? FirstName { get; set; }

        [JsonProperty("lastName")]
        public string? LastName { get; set; }

        [JsonProperty("email")]
        public string? Email { get; set; }

        [JsonProperty("requiredActions")]
        public string[]? RequiredActions { get; set; }

        [JsonProperty("credentials")]
        public IEnumerable<Credentials>? Credentials { get; set; }
    }
}
